dist/linux/debian/changelog

```diff
@@ -1,4 +1,4 @@
-cryptomator (${PPA_VERSION}) bionic; urgency=low
```

Comment:
This change is important and needs to be put into the release notes!

Reply:
Yes. I think Focal is a good choice. I made sure, the [most common Ubuntu-based distros](https://distrowatch.com/search.php?basedon=Ubuntu) don't depend on any _older_ version. So reasonably recent versions are supported:

|               | Based on Focal+ since | Current Release |
|---------------|------------------------------|-----------------|
| Linux Mint    | 20                    | 21              |
| Pop! OS       | 20.04                    | 22.04       |
| Linux Lite    | 5.0                          | 6.2             |
| Zorin OS      | 16                           | 16.2            |
| KDE neon      | 5.20                         | 5.25            |
| elementary OS | 6.0                          | 6.1             |